build_info.cpp

```diff
@@ -154,10 +154,12 @@ const vector<BuildInfo>& BuildInfo::get() {
         "Opens a connection if another portal is present.", 0, "Installations"),
       BuildInfo({FurnitureType::MINION_STATUE, {ResourceId::GOLD, 50}}, "Golden Statue", {},
         "Increases minion population limit by " +
-              toString(ModelBuilder::getStatuePopulationIncrease()) + ".", 0, "Installations"),
+              toString(ModelBuilder::getStatuePopulationIncrease()) + ". (Up to " +
+              toString(ModelBuilder::getStatuePopulationIncrease() * 200) + ")", 0, "Installations"),
```

Comment:
Please define the 4 and 200 numbers in functions in model_builder.cpp similarly to getStatuePopulationIncrease()